build.gradle

```diff
@@ -54,12 +54,6 @@ apply plugin: 'opensearch.opensearchplugin'
 apply plugin: 'opensearch.testclusters'
 apply plugin: 'opensearch.rest-test'
 
-
-checkstyle {
-    toolVersion = '8.24'
-    configFile file("checkstyle/checkstyle.xml")
```

Comment:
why did we hv to remove this?

Reply:
will fix checkstyle  and raise a new revision.